index.html

```diff
@@ -359,12 +359,38 @@ <h3>
       additional members to control audio packetization.
     </p>
     <pre class="idl">partial dictionary RTCRtpEncodingParameters {
-  unsigned long ptime;
-  boolean adaptivePtime = false;
+    RTCResolutionRestriction restrictedResolution;
```

Comment:
bikeshed: resolutionRestriction?
The parameter isn't a resolution, it's a restriction on resolution.

Reply:
Makes sense, done